suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -117,27 +118,29 @@ export const TransactionListItem = memo(
                 onPress={() => handleNavigateToTransactionDetail()}
                 style={applyStyle(transactionListItemStyle, { isFirst, isLast })}
             >
-                <TransactionListItemIcon
-                    cryptoIconName={transaction.symbol}
-                    transactionType={transaction.type}
-                />
-                <Box>
-                    <Box style={applyStyle(titleCointainerStyle)}>
-                        <Text>{transactionTypeProperties.prefix}</Text>
-                        <Box style={applyStyle(addressStyle)}>
-                            <Text
-                                variant="label"
-                                color="gray600"
-                                numberOfLines={1}
-                                ellipsizeMode="middle"
-                            >
-                                {transaction.txid}
-                            </Text>
+                <Box style={applyStyle(descriptionBoxStyle)}>
+                    <TransactionListItemIcon
+                        cryptoIconName={transaction.symbol}
+                        transactionType={transaction.type}
+                    />
+                    <Box marginLeft="medium" flex={1}>
+                        <Box flexDirection="row" justifyContent="center">
+                            <Text>{transactionTypeProperties.prefix}</Text>
+                            <Box style={applyStyle(addressStyle)}>
+                                <Text
+                                    variant="label"
+                                    color="gray600"
+                                    numberOfLines={1}
+                                    ellipsizeMode="middle"
+                                >
+                                    {transaction.targets[0].addresses?.[0]}
```

Comment:
`transaction.targets[0]` could be undefined, so this will crash app in some cases with specific transactions (ERC tokens). Please create selectors `selectTransactionTargets` and `selectTransactionFirstTargetAddress` and handle all possible cases when things could be undefined.

Please use the same selector also for TransactionDetail (which is already crashing app for tokens).

Sorry for bloating this PR but we can't merge something that will crash app even more.

Reply:
Selectors created in commit  [d26ca95](https://github.com/trezor/trezor-suite/pull/7529/commits/d26ca9505d9d5803da0acbf232d1f97b58971c98).